activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -192,12 +192,14 @@ def read_multi(*names)
       #
       # Failsafe: Raises errors.
       def delete_matched(matcher, options = nil)
-        instrument :delete_matched, matcher do
-          unless String === matcher
-            raise ArgumentError, "Only Redis glob strings are supported: #{matcher.inspect}"
-          end
+        unless String === matcher
+          raise ArgumentError, "Only Redis glob strings are supported: #{matcher.inspect}"
+        end
+        options = merged_options(options)
```

Comment:
Is this call to `merged_options` necessary?

Reply:
Ah, you're right, it's not needed at the moment because `namespace_key` will merge the options anyways.